src/CommunityToolkit.Maui.UnitTests/Converters/MathExpressionConverterTests.cs

```diff
@@ -85,9 +124,9 @@ public void MathExpressionConverterNullInputTest()
 		Assert.Throws<ArgumentNullException>(() => ((ICommunityToolkitValueConverter)new MathExpressionConverter()).Convert(0.0, null, "x", null));
 		Assert.Throws<ArgumentNullException>(() => ((ICommunityToolkitValueConverter)new MathExpressionConverter()).ConvertBack(0.0, null, null, null));
 #pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
-		Assert.Throws<ArgumentNullException>(() => ((ICommunityToolkitValueConverter)new MathExpressionConverter()).Convert(null, typeof(bool), "x", null));
+		//Assert.Throws<ArgumentNullException>(() => ((ICommunityToolkitValueConverter)new MathExpressionConverter()).Convert(null, typeof(bool), "x", null));
```

Comment:
Is it possible to amend this Unit Test so that it passes? Or is this Assert failing because of a bug in the new code?

It's not a best-practice to have commented-out assert statements in the Unit Test project.

Reply:
@brminnick the assertion was failing in the new code. I didn't understand the assertion. I will try and figure it out.

Reply:
@brminnick I understand the regression is actually by design by the PR. All values passed thru to the MathExpressionConverters was being sanitized with this logic which raises an exception if any of the inputs are null:

```c#
var valueString = value?.ToString() ?? throw new ArgumentException("Values cannot be null.");
var xValue = double.Parse(valueString);
```

In the new code, I allow nulls, particularly, in the conditional `conditonal ? true_value : false_value` both values on the right can be nullable. In the unit test, it was asking if we past `x=null` to an math expression should it raise an exception. The new behavior it returns null without raising an exception. I believe this to be acceptable improvement, hence, I will rewrite the test to reflect.

Reply:
@brminnick I have amended this unit test